github_connector/demo/github_repository_branch.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+Copyright 2018 Surekha Technologies (https://www.surekhatech.com)
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+<odoo>
+    <!-- Github organization -->
```

Comment:
Only branches with no demo data  for modules?

Reply:
I added the module demo data on the github_connector_odoo